preview/use-cases.md

```diff
@@ -0,0 +1,101 @@
+# Use-cases
+
+## Centralize secret management independent from application
+Differentiating between configuration data and secrets is critical for the security of applications. HTTP ports, application name... can all be seen as public data but access tokens, private keys... should not be in the same group. Accessing them in a centralized manner and secure manner is therefore important.
+
+The [Arcus secret store](https://security.arcus-azure.net/features/secret-store) is the answer. The secret store will be the single point of truth in interacting with required secrets for your application. Multiple 'secret providers' can be configured that accesses the secrets from external sources like: Azure Key Vault, HashiCorp Vault, Docker secrets, environment variables... and the secret store will provide a single access point for the application to fetch those secrets.
+
+An example of how to implement an Azure KeyVault and appsettings config file and how well the secret store integrates within an application:
+
+```csharp
+using Microsoft.Extensions.Hosting;
+
+public class Program
+{
+    public static void Main(string[] args)
+    {
+        CreateHostBuilder(args).Build().Run();
+    }
+
+    public static IHostBuilder CreateHostBuilder(string[] args) =>
+        Host.CreateDefaultBuilder(args)
+            .ConfigureAppConfiguration((context, config) => 
+            {
+                config.AddJsonFile("appsettings.json")
+                      .AddJsonFile("appsettings.Development.json");
+            })
+            .ConfigureSecretStore((context, config, builder) =>
+            {
+#if DEBUG
+                builder.AddConfiguration(config);
+#endif
+                var keyVaultName = config["KeyVault_Name"];
+                builder.AddEnvironmentVariables()
+                       .AddAzureKeyVaultWithManagedServiceIdentity($"https://{keyVaultName}.vault.azure.net");
+            });
+    }
+}
+```
```

Comment:
This code shows how to setup the usage of the secret-store.  Maybe it is interesting to show how to use the secret-store.   Maybe mention that you can just inject some class and you can then easily retrieve secrets from a secretstore.

Reply:
Ah, yes, that would be good! 👍

Reply:
Done in https://github.com/arcus-azure/arcus/pull/204/commits/68128dc124e8affd0a92f6cc4f2c1e5f2fe53dae